packages/x-date-pickers/src/PickersTextField/PickersFilledInput/PickersFilledInput.tsx

```diff
@@ -120,13 +129,13 @@ const PickersFilledInputRoot = styled(PickersInputBaseRoot, {
         },
       },
       {
-        props: ({ startAdornment }: OwnerStateType) => !!startAdornment,
+        props: { isInputAdornedStart: true },
```

Comment:
Super nitty:

```suggestion
        props: { isInputAdornedAtStart: true },
```

Reply:
Absolutely "non-blocking" 🫣

Reply:
I went for `hasStartAdornment` which was proposed by Lukas :+1: